### DISCUSSION TITLE
Transonic tutorial case and finding the learning rate

### DISCUSSION THREAD

**Discussion by firoozeh1** [mdolab/dafoam#346] [Tutorials]
I have troubles with the transonic tutorial case in  NACA0012_Airfoil. The solver didn't reach the optimized point nd I got the message "Maximum Number of Iterations Exceeded." in opt_IPOPT.txt. I was wondering if it relates to the learning rate or the relaxation factor.
How can I change the learning rate in the DAfoam?
I appreciate any suggestions.

**Comment by shaheidula** [comment]
I suppose you need to increase the maximum number of iterations in the runScript. I'm not sure if you have tried this or now.
